src/Components/Loading/Loading.css

```diff
@@ -0,0 +1,28 @@
+.loading-container {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  font-size: xx-large;
+  height: 100%;
+  justify-content: center;
+  width: 100%;
+  z-index: 1;
+}
+
+.loader {
```

Comment:
Thanks for the loader.  Maybe we can find throw in an image to replace the spinner too if we have time?  Maybe a little speaker or something?